eel/src/encryption_symmetric.rs

```diff
@@ -21,7 +23,7 @@ pub(crate) fn encrypt(data: &[u8], key: &[u8]) -> Result<Vec<u8>> {
     Ok(ciphertext)
 }
 
-pub(crate) fn decrypt(data: &[u8], key: &[u8]) -> Result<Vec<u8>> {
+pub(crate) fn decrypt(data: &[u8], key: &[u8; KEY_SIZE_IN_BYTES]) -> Result<Vec<u8>> {
```

Comment:
nit: In this specific case (length of `key`) I would get rid of the constant and use `32` directly. I feel like the constant doesn't add much given that we know that 32 is the key size as it's the array length of an array reference called `key`. A reason to not use it is that the code becomes more concise.

That is my personal preference, but feel free to leave it like this if you disagree.

Reply:
6d9e196a8137532f4202f641af419108b601ca88